### PR TITLE
Update font.eclass to EAPI-7

### PR DESCRIPTION
**Pull request by lovesegfault**
Not to be merged.

cc. @mgorny

### CODE REVIEW

File: eclass/font.eclass [hunk 1]
```diff
@@ -218,7 +218,7 @@ font_pkg_postinst() {
 		elog "The following fontconfig configuration files have been installed:"
 		elog
 		for conffile in "${FONT_CONF[@]}"; do
-			if [[ -e ${EROOT}etc/fonts/conf.avail/$(basename ${conffile}) ]]; then
+			if [[ -e "${EROOT}etc/fonts/conf.avail/$(basename ${conffile})" ]]; then
```
Comment (mgorny) [inline eclass/font.eclass]: Quoting is not needed inside `[[ ... ]]`.

**Comment by mgorny** [comment]
Switching to einstalldocs in EAPI 7 should be beneficial too.

**Comment by ConiKost** [comment]
How about ajusting DOCS to use an array, as other classes do?

**Comment by a17r** [comment]
Let's not duplicate efforts: https://archives.gentoo.org/gentoo-dev/message/c6c178ad1b3da182fc58dd661efb0bc8

**Pull request closed without merging**